packages/yoastseo/babel.config.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
-	presets: [ "@babel/preset-env" ],
+	presets: [
+		[
+			"@babel/preset-env", {
+				modules: false,
```

Comment:
Setting `modules` to `false` is to make sure that the code is not bundled in CommonJS. For treeshaking to work, the module needs to be bundled in ES2015 syntax (from [this source](https://webpack.js.org/guides/tree-shaking/#conclusion) and [this source](https://exploringjs.com/es6/ch_modules.html#sec_advantages-es6-modules))

Reply:
Additionally, I found out that apparently `yarn test` is failing with `SyntaxError: Cannot use import statement outside a module` message when I set `modules` to `false`.

This is because the following:
- Before running test, we're running `yarn build`, where we build ECMAScript modules when `modules` is set to `false` in `babel.config.js`
- We use the build modules for testing
- When running `yarn test`, jest is using CommonJS modules by default. Hence the failing tests

Reply:
Fixed by adjusting the config to still transform the modules to CommonJS when it's for testing. From [this article](https://www.jonathancreamer.com/webpack-tree-shaking-in-3-easy-steps/). Props to @mhkuu for finding it! 🙌🏽